cmd/virtual-kubelet/root/root.go

```diff
@@ -44,6 +44,8 @@ import (
 	"github.com/liqotech/liqo/pkg/virtualKubelet/manager"
 	"github.com/liqotech/liqo/pkg/virtualKubelet/node/module"
 	nodeProvider "github.com/liqotech/liqo/pkg/virtualKubelet/node/provider"
+	nodeprovider "github.com/liqotech/liqo/pkg/virtualKubelet/nodeProvider"
```

Comment:
I suppose one of the two should be dropped

Reply:
no, the first one is vk provided package, with utils functions. The second one is the package containing the liqo node provider (I put it in the same folder where the liqo pod provider package was)

Reply:
Ok, then I would rename the latter to liqoNodeProvider, otherwise it is very hard to distinguish the two